governance/remote_executor/cli/src/main.rs

```diff
@@ -167,9 +169,18 @@ fn main() -> Result<()> {
             let wormhole_config_data =
                 Config::try_from_slice(&rpc_client.get_account_data(&wormhole_config)?)?;
 
+            let executor_key = Pubkey::find_program_address(
+                &[EXECUTOR_KEY_SEED.as_bytes(), &payer.pubkey().to_bytes()],
+                &ID,
+            )
+            .0;
             let payload = ExecutorPayload {
                 header:       GovernanceHeader::executor_governance_header(cli.chain),
-                instructions: vec![],
+                instructions: vec![InstructionData::from(&system_instruction::transfer(
```

Comment:
I added an instruction here in the test vaa to give us more confidence. It transfer 1 lamport from  one of the executor keys.